### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by rainrivas**
Currently published v0.2.0 as slng@beta on npm.
Once PR is merged, I'll republish and tag as slng@latest

## Fixes
* #5 - Add colors to output
* #7 - No output when results not found
* #8 - Add version flag, `-V, --version`
* #10 - Help doesn't print a clear usage message
* #12 - Add contribution guidelines
* #13 - Add more relevant tags to package.json (keywords)

### CODE REVIEW

**Comment by rainrivas** [comment]
Should we be merging this with merge commit or squash? Thoughts?

File: index.js [hunk 1]
```diff
@@ -22,16 +24,21 @@ program
...
-          console.log("=======================\n" + result.definition);
+      else {
+        if (results.length > 3) {
+          trimRes = results.slice(0, 3);
+        } else {
+          trimRes = results;
         }
-      });
+        trimRes.forEach((result) => {
+          if (typeof (result.definition) !== undefined) {
```
Comment (jwu910) [inline index.js:37]: Is there a case that this would be undefined?

**Comment by jwu910** [comment]
@rainrivas I think we can merge w/o squash.

**Comment by rainrivas** [comment]
@jwu910 so we will merge commit right? (I am pretty sure you don't rebase commits if it's a release)

**Comment by jwu910** [comment]
@rainrivas Correct. I can agree with that. Merge commit for releases.

**Review by jwu910 — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (rainrivas) [reply index.js:37]: @jwu910 if there is a bad response and the response doesn't have a "definition" element. It's sort of a catch all. For example
```
{
    "tags": [],
    "result_type": "no_results",
    "list": [],
    "sounds": []
}
```
vs actually finding results
```
{
    "tags": [
        "sleng",
        "colloquialism",
        "neologism",
        "slang"
    ],
    "result_type": "exact",
    "list": [
        {
            "definition": "SLNG is actually slang for...slang.",
            "permalink": "http://slng.urbanup.com/1146042",
            "thumbs_up": 9,
            "author": "DaWhitey",
            "word": "slng",
            "defid": 1146042,
            "current_vote": "",
            "example": "Mah is slng for My \"Ur slng so cheep\"",
            "thumbs_down": 4
        }
    ],
    "sounds": []
}
```

**Pull request merged**